react-exercise/src/components/user-table/UserTable.scss

```diff
@@ -1,3 +1,26 @@
+.user-table-container {
+  overflow-y: auto;
+  max-height: 680px;
+
+  /* Custom Scrollbar */
+  ::-webkit-scrollbar {
```

Comment:
Are these definitions actually needed? Or did GPT suggest them ?

Reply:
I tried to look how to style the scroller but it actually doesn't work :(
If you know how I can style it I will like to hear

Reply:
Maybe try to add & before?

"&::-webkit-scrollbar {"

Reply:
yeessss thank you I missed it!